gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+              <NxTable.Cell>Highlights the row to show that the list item is selected.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>disabled</NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>Whether the list item should be rendered as disabled or not.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>HTML <NxCode>{'<button>'}</NxCode> attributes</NxTable.Cell>
+              <NxTable.Cell>
+                <NxTextLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button" external>
+                  HTML button attributes
+                </NxTextLink>
+              </NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>NxList.Button supports any HTML attribute that's normally supported
+                by <NxCode>{'<button>'}</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>NxList.Link</NxTable.Cell>
+              <NxTable.Cell>href</NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>yes</NxTable.Cell>
+              <NxTable.Cell>Link to the URL provided.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>selected</NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>Highlights the row to show that the list item is selected.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>disabled</NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>Whether the list item should be rendered as disabled or not.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell> </NxTable.Cell>
+              <NxTable.Cell>HTML <NxCode>{'<a>'}</NxCode> attributes</NxTable.Cell>
+              <NxTable.Cell>
+                <NxTextLink href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a" external>
+                  HTML anchor attributes
+                </NxTextLink>
+              </NxTable.Cell>
+              <NxTable.Cell>no</NxTable.Cell>
+              <NxTable.Cell>NxList.Link supports any HTML attribute that's normally supported
+                by <NxCode>{'<a>'}</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>NxList.Text</NxTable.Cell>
+              <NxTable.Cell>truncate</NxTable.Cell>
```

Comment:
We don't typically encode this one as a prop but rather let it be applied as a CSS class where needed.

Reply:
> We don't typically encode this one as a prop but rather let it be applied as a CSS class where needed.

The `truncate` prop has been removed. Need to use class `nx-truncate-ellipsis` for truncation now. 
d0e7d2dae13a5d1264d1c443cda55b34cb30525e